.changeset/good-peas-type.md

```diff
@@ -0,0 +1,5 @@
+---
+'@celo/celocli': minor
```

Comment:
id say this would be a major as its removing something

Reply:
Yep good catch